misk/src/main/kotlin/misk/client/HttpClientFactory.kt

```diff
@@ -56,10 +55,12 @@ class HttpClientFactory @Inject constructor(
     }
 
     config.clientConfig.protocols?.let {
-      builder.protocols(
-          config.clientConfig.protocols.map { Protocol.get(it) }
-              ?: listOf(Protocol.HTTP_1_1) //Safe default
```

Comment:
This is a bug that already exists. `?:` only evaluates for nullable types, which the result of line 60 is not.

Reply:
I have fixed this in my other PR (currently in review).

Reply:
Although, given that your PR does not have any other changes in this file, I think it's better to not commit this. This is not actually a bug, just a leftover from refactoring that I forgot to remove. The `protocols` will never be empty, it's either null or has something in it.